src/app/common/CopyButton.vue

```diff
@@ -94,23 +94,22 @@ async function copy(event: Event, copyToClipboard: (text: string) => Promise<boo
 }
 
 .copy-button {
-  --tooltip-text-color: var(--white);
-  --tooltip-background-color: var(--grey-600);
+  --tooltip-background-color: $kui-color-background-neutral-stronger;
 }
 
 .copy-button[data-tooltip-copy-success="false"] {
-  --tooltip-background-color: var(--red-700);
+  --tooltip-background-color: $kui-color-background-danger-stronger;
 }
 
 .copy-button[data-tooltip-text]::after {
   background-color: var(--tooltip-background-color);
   border-radius: 3px;
-  color: var(--tooltip-text-color);
+  color: #fff;
```

Comment:
nit
```suggestion
  color: $kui-color-text-inverse;
```

Reply:
Applied this in a few applicable cases in https://github.com/kumahq/kuma-gui/pull/1314/commits/0574d89b431f05cdbd2c3cb4653e425c52f6d1d0.